package.json

```diff
@@ -27,6 +28,7 @@
   "license": "MIT",
   "dependencies": {
     "@babel/polyfill": "^7.0.0",
+    "commander": "^2.18.0",
```

Comment:
why not yargs?

Reply:
I don't the style of `yargs` API, and I know more about `commander`.

Reply:
we should get the opinions of babel maintainers and their preferences, since that’s who will be maintaining it.

Reply:
We are already using commander for babel-cli, so using commander here too seems fine. https://github.com/babel/babel/blob/master/packages/babel-cli/package.json#L22